src/components/country/CountryDetail.jsx

```diff
@@ -0,0 +1,94 @@
+import { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { searchByCode } from '../../redux/countries/countriesAction';
+import { reset } from '../../redux/countries/countriesSlice';
+import './CountryDetail.css';
+
+const CountryDetail = () => {
+  const { error, countrySearched } = useSelector((state) => state.country);
+  const dispatch = useDispatch();
+  const { code } = useParams();
+  useEffect(() => {
+    if (code) {
+      dispatch(searchByCode(code.toLowerCase()));
+    }
+
+    if (error) {
+      dispatch(error);
+    }
+    return () => {
+      dispatch(reset());
+    };
+  }, [dispatch, code, error]);
+  return (
+    <section className="country-detail-container">
+      <div className="country-detail-content">
+        {countrySearched.length > 0 ? (
+          <>
+            <img
+              src={countrySearched[0].coatOfArms.png}
+              alt={countrySearched[0].coatOfArms.alt}
+              className="country-detail-image"
+            />
+
+            <div className="country-detail-right">
+              <h1>{countrySearched[0].name.common}</h1>
+              <div className="details">
+                <div className="detail-left">
+                  <p>
+                    Offcial Name:
+                    {' '}
+                    <span>{countrySearched[0].name.official}</span>
+                  </p>
+                  <p>
+                    Population:
+                    {' '}
+                    <span>{countrySearched[0].population}</span>
+                  </p>
+                  <p>
+                    Region:
+                    {' '}
+                    <span>{countrySearched[0].region}</span>
+                  </p>
+
+                  <p>
+                    Time Zone:
+                    {' '}
+                    <span>{countrySearched[0].timezones[0]}</span>
+                  </p>
+                  <p>
+                    Capital:
+                    {' '}
+                    <span>{countrySearched[0].capital}</span>
+                  </p>
+                </div>
+
+                <div className="detail-right">
+                  <p>
```

Comment:
- [ ] It would be nice if you could modify your styles for the details page to match the original design as required. You could design a single detail from a list of metrics details for a single country to be in a card look, spacing the item name at the left and the number at the right. See the screenshots below as I wrapped your design with a border on a single detail. 

**YOUR DESIGN**

<img width="452" alt="Screenshot 2023-04-18 at 10 48 36 AM" src="https://user-images.githubusercontent.com/50941074/232762744-4405ff69-cd8e-46a8-a916-3b29fcd5dcef.png">

**THE ORIGINAL DESIGN**

<img width="407" alt="Screenshot 2023-04-18 at 10 50 22 AM" src="https://user-images.githubusercontent.com/50941074/232762850-322ec090-ba20-443e-a162-43ebc03a644b.png">